src/dashboard/src/templates/administration/sidebar.html

```
@@ -36,6 +37,8 @@
 
     <li class="{% active request url_atk_dips %}"><a href="{{ url_atk_dips }}">Archivists Toolkit DIP upload</a></li>
 
+    <li class="{% active request url_netx %}"><a href="{{ url_netx }}">NetX</a></li>
```

Maybe hide this by default, the way you can hide the AtoM DIP upload link?